packages/core/saved-objects/core-saved-objects-import-export-server-internal/src/import/lib/collect_saved_objects.test.ts

```diff
@@ -139,8 +148,8 @@ describe('collectSavedObjects()', () => {
       const result = await collectSavedObjects({ readStream, supportedTypes, objectLimit });
 
       const collectedObjects = [
-        { ...obj1, typeMigrationVersion: '' },
-        { ...obj2, typeMigrationVersion: '' },
+        { ...obj1, typeMigrationVersion: '', managed: false },
```

Comment:
`collectObjects` calls `createSavedObjects` that calls `savedObjectsClient.bulkCreate`, which will set the default if not already specified on the doc.